Packs/rasterize/Integrations/rasterize/rasterize.yml

```diff
@@ -217,7 +217,7 @@ script:
     description: Converts a PDF file to an image file.
     execution: false
     name: rasterize-pdf
-  dockerimage: demisto/chromium:1.0.0.27341
+  dockerimage: devdemisto/chromium:1.0.0.27660
```

Comment:
This is not the correct one. I'll change it once the docker pr will be merged, and we will get the correct docker image.